### ISSUE TITLE
Directional shadow framebuffer status invalid error and slow start [Windows 7, Radeon HD 3650]

### ISSUE THREAD

**Issue by OlexiyKravchuk** [godotengine/godot#27572]
**Godot version:**
Godot_v3.1-stable_win64.exe

**OS/device including version:**
ОС: Windows 7 x64 
Radeon HD 3650

**Issue description:**
Every time I launch the project manager, I get a bunch of errors, 30-31x the same about rasterization, and 1x about Directional shadow framebuffer status invalid. Why does the project manager need a shadow buffer?
![image](https://user-images.githubusercontent.com/10380658/55299029-a46d3400-5439-11e9-881e-650c3d983724.png)

**Comment by qarmin** [comment]
Have you installed the latest drivers from AMD? - https://www.amd.com/en/support/graphics/amd-radeon-hd/ati-radeon-hd-3000-series/ati-radeon-hd-3650

Probably Godot create some basic objects, which are used in editor and which require this functions.

**Comment by OlexiyKravchuk** [comment]
> qarmin

yes, I did it first thing.

**Comment by qarmin** [comment]
This doesn't seems to be Godot issue, but poorly OpenGL support in older drivers.
As a workaround I suggest use Mesa opengl version(read latest comments - https://github.com/godotengine/godot/issues/25607) or just use Linux which have Mesa library built-in.

**Comment by Zireael07** [comment]
I believe the issue is not the error itself (it's known to happen across a variety of devices and already reported several times) - but the question is why does Project Manager spew it. Unlike when the project is loaded, the project list itself uses no shadows at all...

**Comment by Calinou** [comment]
@Zireael07 The project manager uses 3D framebuffer allocation as it's the default value. Maybe making it use 2D framebuffer allocation would solve this (while also making it slightly faster on slow GPUs).

**Comment by akien-mga** [comment]
The project manager uses the same renderer as the engine. Even without using 3D features, it does allocate the required framebuffers for the renderer, as Calinou mentioned.

But that's beside the point. If this error is raised on the project manager, it's raised on the editor too, and on exported games.

**Comment by akien-mga** [comment]
>  As a workaround I suggest use Mesa opengl version(read latest comments - #25607) or just use Linux which have Mesa library built-in.

Using MesaGL for Windows is probably your best shot indeed.

**Comment by OlexiyKravchuk** [comment]
> akien-mga

You did not correctly interpret this problem, the name does not match, if you look closely at the number of errors, almost all of them are No. 4996. and most importantly, the project manager opens FAST in GLES3 mode. And in the problem https://github.com/godotengine/godot/issues/27584, I reported SLOWLY launching projects and opening it in the editor in GLES2 mode.

**Comment by rodolforg** [comment]
Same problem here on Windows 10 Pro.
```
Running: C:\Users\1556897\Desktop\Godot_v3.1.1-stable_win64.exe --path C:/Users/1556897/Documents/teste --remote-debug 127.0.0.1:6007 --allow_focus_steal_pid 10160 --position 448,240 res://Node2D.tscn
OpenGL ES 2.0 Renderer: ATI Radeon HD 4250
ERROR: initialize: Directional shadow framebuffer status invalid
   At: drivers/gles2/rasterizer_scene_gles2.cpp:3369
```

**Comment by rodolforg** [comment]
@akien-mga  you mean this? https://fdossena.com/?p=mesa/index.frag

Didn't work here :/

**Comment by qarmin** [comment]
>> Note: MesaInjector is currently incompatible with Windows 10 1803, it will work again as soon as Thinapp gets updated

**Comment by Calinou** [comment]
@rodolforg Try installing https://github.com/pal1000/mesa-dist-win (you can enable it just for Godot by placing the DLL files in the same directory as the Godot editor executable).

**Comment by rodolforg** [comment]
@qarmin  If that comment/quote was directed to me, it's just for Solution 2. I tried solution "0" and 1.

@Calinou Tried, but sadly it does not even start. 
```
c:\Users\1556897\Desktop\teste 31>Godot_v3.1-stable_win64.exe
OpenGL ES 3.0 Renderer: llvmpipe (LLVM 8.0, 128 bits)

c:\Users\1556897\Desktop\teste 31>
```
And it closes.

**Comment by Skaruts** [comment]
@Calinou 
> Try installing https://github.com/pal1000/mesa-dist-win;

I tried that, and ran it's per app deployment, but after asking me for the path and executable, it also asks me some stuff that I've no clue what to choose:
```
"Do you want Desktop OpenGL drivers (y/n, defaults to yes):"
"Do you need off-screen rendering (y/n):"
"Do you need graw library (y/n):"
```
Instructions on their page don't mentioned any of this.

**Comment by Calinou** [comment]
@Skaruts This is a software OpenGL implementation, so it's going to be slow no matter your hardware :slightly_frowning_face:

**Comment by akien-mga** [comment]
Should be fixed by #31259.

Could you test to confirm?
